scripts/update-dockerfiles.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -o pipefail
+set -e
+
+function update_dockerfile() {
+    local image=$1
+    local image_name=$(echo $image | awk -F/ '{print $NF}' | awk -F: '{print $1}')
+
+    [[ -z $image_name ]] && { echo "image_name is empty for image: $image">&2; exit 1; }
+    mkdir -p images/$image_name
+    if [[ ! -f images/$image_name/Dockerfile ]];
+        then
+            echo "FROM $image" > images/$image_name/Dockerfile
+        else
+            sed -i "s|FROM .*$image_name.*|FROM $image|" images/$image_name/Dockerfile
+        fi
+}
+
+
+function with_helm() {
+    helm template . | awk '/^[ \t"-]*image["]*: [a-zA-Z0-9/:@"\.-]+$/{print $NF}' | sed 's/"//g' | \
```

Comment:
This wont work, because we have to replace all the images with ours in the helm chart, the output of `helm template .` will always return our values. Try parsing upstream helm chart instead